curl-sys/build.rs

```diff
@@ -30,7 +31,8 @@ fn main() {
     // Next, fall back and try to use pkg-config if its available.
     match pkg_config::find_library("libcurl") {
         Ok(..) => return,
-        Err(..) => {}
+        Err(e) => writeln!(&mut stderr(), "Couldn't find libcurl from \
+                           pkgconfig ({:?}), compiling it from source...", e).unwrap(),
```

Comment:
Ah nah it's fine to dump all other kinds of data to stdout so long as it doesn't look like "cargo:foo=bar"